data-raw/DATASET.R

```diff
@@ -1,8 +1,15 @@
 ## code to prepare `DATASET` dataset goes here
 
-ADSL <- haven::read_xpt("https://github.com/cdisc-org/sdtm-adam-pilot-project/raw/master/updated-pilot-submission-package/900172/m5/datasets/cdiscpilot01/analysis/adam/datasets/adsl.xpt")
-ADAE <- haven::read_xpt("https://github.com/cdisc-org/sdtm-adam-pilot-project/raw/master/updated-pilot-submission-package/900172/m5/datasets/cdiscpilot01/analysis/adam/datasets/adae.xpt")
-ADTTE <- haven::read_xpt("https://github.com/cdisc-org/sdtm-adam-pilot-project/raw/master/updated-pilot-submission-package/900172/m5/datasets/cdiscpilot01/analysis/adam/datasets/adtte.xpt")
+ADSL <- haven::read_xpt(
+  "https://github.com/cdisc-org/sdtm-adam-pilot-project/raw/master/updated-pilot-submission-package/900172/m5/datasets/cdiscpilot01/analysis/adam/datasets/adsl.xpt"
+)
+ADAE <- haven::read_xpt(
+  "https://github.com/cdisc-org/sdtm-adam-pilot-project/raw/master/updated-pilot-submission-package/900172/m5/datasets/cdiscpilot01/analysis/adam/datasets/adae.xpt"
+)
+ADTTE <- haven::read_xpt(
+  "https://github.com/cdisc-org/sdtm-adam-pilot-project/raw/master/updated-pilot-submission-package/900172/m5/datasets/cdiscpilot01/analysis/adam/datasets/adtte.xpt"
+)
 
+ADSL |> mutate(TRTA = ARM)
```

Comment:
Thinking about this on a Friday afternoon...let's update line to assign actual treatment to actual treatment.

```r
ADSL$TRTA <- ADSL$TRT01A
```

Reply:
Then we'll want to add a column label that matches the label in ADAE